content/en/news/releases/1.15.x/announcing-1.15.6/index.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Announcing Istio 1.15.6
+linktitle: 1.15.6
+subtitle: Patch Release
+description: Istio 1.15.6 patch release.
+publishdate: 2023-02-21T07:00:00-06:00
+release: 1.15.6
+aliases:
+    - /news/announcing-1.15.6
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.15.5 and Istio 1.15.6.
```

Comment:
Nit: Should we remove the first sentence?

Reply:
I wasn't sure how to word the release note when there were no changes other than Go.